egs/aishell/s10b/local/token_to_fst.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Mobvoi AI Lab, Beijing, China (author: Fangjun Kuang)
+# Apache 2.0
+
+import argparse
+import os
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='convert tokens.txt to tokens.fst')
+
+    parser.add_argument('--tokens-txt-filename',
```

Comment:
Please clarify what the input is.  I.e. what the format of tokens.txt is.

Reply:
Fixed in the newly added documentation.